docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -24,21 +24,21 @@ Clusters configured using the following options can be selected when deploying f
 
 To add additional clusters, increment the `0` value for each variable (`CAMUNDA_MODELER_CLUSTERS_1_NAME`).
 
-| Environment variable | Description | Example value |
```

Comment:
I'm not really sure how made it in, as it should have been caught by basically every run of `npm run format:check` 🤷